contracts/SmartVaultYieldManager.sol

```diff
@@ -151,13 +153,14 @@ contract SmartVaultYieldManager is ISmartVaultYieldManager, Ownable {
 
     function deposit(address _collateralToken, uint256 _euroPercentage) external returns (address _hypervisor0, address _hypervisor1) {
         // TODO min _euroPercentage of 10%
```

Comment:
why don't we add the require statement as pointed out in the Todo?

Reply:
that's done now, was not yet finished developing that out